### PR TITLE
Legislation improvements

### PR DESCRIPTION
**Pull request by actlikewill**
Makes the listing page title configurable in the template
Makes the legislation tab titles configurable in the template
Includes citation as a subheading below main heading in document detail page in gray text
Includes breadcrumbs in detail pages (judgments)
Adds alphabetical filter to legislation

image
image
image

closes laws-africa/kenyalaw-pj#55
closes laws-africa/kenyalaw-pj#66
closes laws-africa/kenyalaw-pj#74

### CODE REVIEW

File: liiweb/views/legislation.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ def filter_queryset(self, qs):
             qs = qs.exclude(parent_work=None).filter(
                 repealed=False, metadata_json__principal=True
             )
+        elif self.variant == "uncommenced":
+            qs = qs.filter(metadata_json__commencement_date=None)
```
Comment (longhotsummer) [inline liiweb/views/legislation.py:41]:
```suggestion
            qs = qs.filter(metadata_json__commenced=False)
```

File: peachjam/models/generic_document.py [hunk 2]
```diff
@@ -103,22 +103,40 @@ def search_penalty(self):
             return 10.0
         return super().search_penalty()
 
+    @property
+    def commenced(self):
+        return bool(self.metadata_json.get("commencement_date", None))
```
Comment (longhotsummer) [inline peachjam/models/generic_document.py:108]:
```suggestion
        return bool(self.metadata_json.get("commenced", None))
```

File: peachjam/templates/peachjam/legislation_detail.html [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {% extends 'peachjam/layouts/document_detail.html' %}
 {% load i18n peachjam %}
+{% block sub-title %}<h2 class="h5 text-muted">{{ document.citation }}</h2>{% endblock %}
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/legislation_detail.html:3]: Only if `document.title != document.citation` which is often true for judgments. Also guard against no citation.

**Comment by longhotsummer** [comment]
Nice!

**Comment by actlikewill** [comment]
@longhotsummer I haven't added labels to the legislation listing, should I include them here or could you add them with your changes to the VueJS part?

**Comment by longhotsummer** [comment]
> @longhotsummer I haven't added labels to the legislation listing, should I include them here or could you add them with your changes to the VueJS part?

I'll pull them in.

**Review by longhotsummer — DISMISSED** [review]

**Review by longhotsummer — APPROVED** [review]

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**